docs/self-managed/console-deployment/telemetry.md

```diff
@@ -0,0 +1,115 @@
+---
+id: telemetry
+title: "Console (Self-Managed) Telemetry"
+sidebar_label: "Telemetry"
+description: "Console (Self-Managed) Telemetry introduction"
+---
+
+## Camunda 8 Console Self-managed Telemetry
+
+At Camunda, we strive to offer an excellent user experience at a high and stable level. On a strictly opt-in basis, we are looking to collect environment and usage data to improve your user experience further. These insights help us understand typical environment setups and product usage patterns and will be used to make informed product improvement decisions that benefit you.
+
+### Design
+
+The Camunda 8 Console self-managed incorporates a telemetry mechanism designed to periodically report usage data to Camunda. This telemetry operates seamlessly in the background and is activated only when telemetry is explicitly enabled. The data collection mechanism is structured to function without interrupting or degrading the performance of the implemented processes. When enabled, the collected data is sent once every 24 hours via HTTPS.
+
+Once telemetry is enabled, a new Telemetry page will display collected information. This page can be used to view Component metrics as a simple monitoring interface. The Console will collect a subset of metrics, which you can download and inspect anytime.
+
+### Collected Data
+
+The telemetry feature categorizes the collected data into general Camunda data and metrics data:
+
+- General Data: This includes information about deployed Camunda components, customer ID, and unique installation identifier.
+- Metrics Data: Captures selected Prometheus metrics available for each component via monitoring endpoint. For example, command counts, metrics related to process instances, CPU load, and memory allocation.
+
+### How to Enable Telemetry
+
+By default, the Telemetry configuration is set to “disabled.” Telemetry can be activated by setting the appropriate configuration in the Camunda 8 Helm Chart or setting the appropriate environment variables in the Console configuration. Please refer to the [Console configuration](./configuration.md) page for more information.
+
+### Telemetry configuration options explained.
+
+When the telemetry configuration is set to `disabled`, the Console will not collect or send any metrics. This feature will not be running.
```

Comment:
Maybe worth adding that this is the default setting. If any other or no value is configured, telemetry is disabled.